contrib/ansible/create-cluster-playbook.yaml

```diff
@@ -48,6 +48,17 @@
   - import_role:
       name: kubectl-ansible
 
+  - name: process cluster versions template
+    oc_process:
+      template_file: "{{ playbook_dir }}/../examples/cluster-versions-template.yaml"
+    parameters:
+      CLUSTER_VERSION_NS: "{{ cluster_version_namespace }}"
+    register: cluster_versions_reg
+
+  - name: create/update cluster versions
+    kubectl_apply:
+      definition: "{{ cluster_versions_reg.result | to_json }}"
```

Comment:
Couldn't do this in the deploy playbook anymore as we don't necessarily have a running deployment after it completes, you need to push images or start-build.